lib/fs.js

```diff
@@ -1867,6 +1867,49 @@ WriteStream.prototype._write = function(data, encoding, cb) {
 };
 
 
+function writev(fd, chunks, position, callback) {
+  function wrapper(err, written) {
+    // Retain a reference to chunks so that they can't be GC'ed too soon.
```

Comment:
Yes, that is correct (afaik). Buffers allocate their own memory and delimit deallocating to v8, which doesn't take into account the event loop. Relevant: https://github.com/nodejs/io.js/commit/f9b7714b4b3f04cba10c4d8b32dad48a68988799